### DISCUSSION TITLE
Upscayl ComfyUI node or workflow?

### DISCUSSION THREAD

**Discussion by joshgura** [upscayl/upscayl#955] [Ideas]
Wouldn't it be cool if there was an Upscayl node or workflow in Comfy? Something like SUPIR. This would generate a lot of buzz. Even if it was only a workflow that didn't use a custom Upscayl node, it would be a nice showcase for the expertise of the Upscayl devs, and generate some buzz.

**Comment by aaronliu0130** [comment]
I don't really see the purpose of investing the resources into maintaining something that's basically useless when you consider the REAL-ESRGAN node, which effectively powers our app.
